### DISCUSSION TITLE
Welcome to awesome-repos Discussions!

### DISCUSSION THREAD

**Discussion by pawelborkar** [pawelborkar/awesome-repos#5] [Announcements]
## 👋 Welcome to Awesome Repos!
> A curated list of GitHub Repositories full of FREE Resources.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Coady533** [comment]
I'm Musa an expert in both HTML and CSS
